src/main/java/cz/muni/fi/pv286/mmt/model/Options.java

```diff
@@ -0,0 +1,163 @@
+package cz.muni.fi.pv286.mmt.model;
+
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.Optional;
+
+/**
+ * This class encapsulates all possible options.
+ */
+public class Options {
+    private IoFormat inputFormat;
+    private Optional<IoOption> inputFromToOption = Optional.empty();
+    private IoFormat outputFormat;
+    private Optional<IoOption> outputFromToOption = Optional.empty();
```

Comment:
**question:** Why did you use Optional<>? 

**note:** This can be checked on null.